services/management/commands/update_vantaa_parking_areas.py

```diff
@@ -63,6 +76,7 @@
     "Maksullinen": {"sv": "Avgiftsbelagd", "en": "Paid"},
     "Muu": {"sv": "Något annat", "en": "Other"},
     "Varattu päivisin": {"sv": "Bokas dagtid", "en": "Reserved during the day"},
+    "Liityntäpysäköinti": {"sv": "Pendelparkering", "en": "Park & Ride"},
```

Comment:
Should these be in `*.po`-translations?

Reply:
These are translations for data and not to any literal in the code, so I don't think it is possible?

Reply:
Ok, we can skip these then.